StandardLibraryEssentials/Templates/TypeInference/task.md

```diff
@@ -0,0 +1,49 @@
+Type inference is a powerful feature in C++ that allows the compiler to automatically deduce the type of a variable or expression at compile time. This can make your code more concise and easier to read, as well as reduce the likelihood of type-related errors. When dealing with types with complex and obscure names, type inference can make your code more readable and maintainable.
```

Comment:
```suggestion
Type inference is a powerful feature in C++ that allows the compiler to automatically deduce the type of a variable or an expression at compile time. This can make your code more concise and easier to read, as well as reduce the likelihood of type-related errors. When dealing with types with complex and obscure names, type inference can make your code more readable and maintainable.
```